backend/src/main/scala/petclinic/server/ClinicServer.scala

```diff
@@ -34,17 +50,24 @@ final case class ClinicServer(
         }
     }
 
+  /** Resets the database to the initial state every 15 minutes to clean up the
+    * deployed Heroku data. Then, it obtains a port from the environment on
+    * which to start the server. In the case of being run in production, the
+    * port will be provided by Heroku, otherwise the port will be 8080. The
+    * server is then started on the given port with the routes provided.
+    */
```

Comment:
```suggestion
  /** Forks a fiber that resets database to the initial state every 15 minutes to clean up the
    * deployed Heroku data. 
    *
    * Then, it obtains a port from the environment on which to start the server. 
    * In the case of being run in production, the port will be provided by Heroku, 
    * otherwise the port will be 8080. The server is then started on the given port 
    * with the routes provided.
    */
```